lib/provision.rb

```diff
@@ -39,7 +44,12 @@ def setup_build
 
     def setup_conveniences
       executer = Executer.new("data/setup")
+
+      #setup folder where registry hold docker images
+      executer.run_in_vm!("sudo mkdir -p /atlantis-docker")
+
       # Set up nice ssh access
+      
```

Comment:
Nitpick, but a blank line between the comment and the code its commenting seems a bit strange.